### ISSUE TITLE
Discontinuities in acceleration for clamped problem run with Schwarz 

### ISSUE THREAD

**Issue by ikalash** [lxmota/Norma.jl#17]
In setting up / running the clamped problem with Schwarz, I have discovered discontinuities in the acceleration field (see screenshots below).  It appears this issue actually shows up only in the initial time-step, but, curiously, there are noticeable errors in the acceleration field in subsequent time-steps (see screenshots below), suggesting Schwarz may not be converged.  The tolerance is pretty tight already: https://github.com/lxmota/Norma.jl/blob/main/examples/ahead/overlap/clamped/dynamic/clamped.yaml#L11-L12 .  I can try tightening it to see if it helps.  I wonder if the acceleration should go into the convergence criterion?  I can try adding that too to see if it helps.

It seems the issue with the discontinuity may have to do with the interpolation of the acceleration data not happening at the right time relative to the acceleration solve (https://github.com/lxmota/Norma.jl/blob/main/src/time_integrator.jl#L193).

![soln_0001](https://github.com/user-attachments/assets/ee2279a7-ce17-42ed-a325-e8150fcb3ceb)
![soln_0001_zoom](https://github.com/user-attachments/assets/d68bee40-4b8c-4e08-84e8-e0bb92665902)
![soln_0003](https://github.com/user-attachments/assets/bb56abc4-242a-49c5-8458-dfaca53ed41b)
![soln_0003_zoom](https://github.com/user-attachments/assets/472550e0-6787-4ed9-971e-489bc670ff71)

**Comment by ikalash** [comment]
Curiously, the oscillations in acceleration near the Schwarz boundary do not lessen when I change the convergence criterion to include acceleration or reduce the Schwarz tolerance to as low as 1e-20.  This was surprising.

**Comment by ikalash** [comment]
Also, I just verified that this happens with both implicit and explicit time-steppers.

**Comment by ikalash** [comment]
I have looked into this more.  The discontinuity in the initial acceleration is a feature of the relative order of operations in which things happen.  The code works as follows: ICs are set in each subdomain, BCs are applied in subdomain 1, an initial solve for the acceleration is done in subdomain 1, BCs are applied in subdomain 2, an initial solve for the acceleration is done in subdomain 2, and the initial data are written to csv file.  After this the Schwarz iteration proceeds.  When the initial solution is set in Omega1, there is no data yet from Omega2.  Hence, the initial acceleration is 0 at the Schwarz boundary.  I have verified that the accelerations match in subsequent Schwarz iterations.

Regarding the oscillations in acceleration near the Schwarz boundary, these appear to be Gibbs oscillations coming from the mesh being too coarse.  I have verified that these go away if you refine the mesh.  

Closing this issue.
